### PR TITLE
CUI support install from NPM

### PR DESCRIPTION
**Pull request by awinny**
- Changed gulp setup
- Changed dependency handling
- Changed build step to use prepublish instead of prepare making it so that this scrip is only executed when running npm i and not npm install corporate-ui
- Changed template files to reflect the new routes

**TODO - Fix font-awesome dependency handling**

### CODE REVIEW

**Comment by helloimela** [comment]
I have tested on Firefox, IE, and Chrome, and now it works perfectly fine :)
The problem is only font-awesome in the corporate-ui-light version. And it is because we use font-awesome "light" which is not included in the free version. When we installed font-awesome dependency from npm, it includes the free version only which does not have the fa-light fonts. That's why it does not render a correct icon in the corporate-ui-light version. So maybe we should include OUR font-awesome inside corporate-ui dependency.  The files that are missing are **fa-light-300** font files.

File: src/global/less/corporate-ui-light.less [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+/*!
+ * Corporate UI
+ * Maintained and developed by IXCC : corporate-ui.slack.com
+ */
+
+// Insert Compile Date
+@compiled_at: `new Date()`;
+.compiled_at {content: "@{compiled_at}";}
+
```
Comment (helloimela) [inline src/global/less/corporate-ui-light.less]: Probably add fontawesome in corporate-ui-light.less or somewhere else to support "fal fa-" in cui-light
@import url("https://static.scania.com/resources/icons/font-awesome/5.3.1/css/all.min.css");

File: package.json [hunk 2]
```diff
@@ -29,27 +32,31 @@
...
     "less": "2.7.2",
+    "lobipanel": "1.0.0",
     "merge-stream": "1.0.1",
     "moment": "2.19.3",
     "react": "15.4.2",
     "react-datetime": "2.8.8",
     "react-dom": "15.4.2",
     "redux": "1.0.0",
     "redux-watch": "1.1.1",
+    "terser-webpack-plugin": "^1.2.1",
```
Comment (awinny) [inline package.json]: remove caret symbol

File: src/components/dashboard/style.less [hunk 3]
```diff
@@ -1,5 +1,7 @@
 @import (reference, less) "../../../node_modules/bootstrap/dist/css/bootstrap.css";
 
+@import '~lobipanel/dist/css/lobipanel.css';
```
Comment (awinny) [inline src/components/dashboard/style.less]: remove tilde symbol

**Comment by mmexvr** [comment]
Since this is a completly new feature we

**Pull request closed without merging**